src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx

```diff
@@ -217,7 +217,9 @@ export const CreateNSTreeModal = ({
           phylo_run_workflow_id: respData.id,
           // Safe to assert treeType is not undefined here, can't create tree
           // otherwise, it's just that checking happens in a child component.
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
why is this override needed now, when it was working before?

Reply:
There was always a warning in the list of things we generally ignore.  This just makes the warning go away and seems ok based on the comment above.